packages/ember-auto-import/ts/webpack.ts

```diff
@@ -172,7 +179,9 @@ export default class WebpackBundler extends Plugin implements Bundler {
         ),
       },
       module: {
-        noParse: (file: string) => file === join(stagingDir, 'l.js'),
+        // I guess webpack should also be prevented from parsing our virtual modules, so presumably we could
+        // filter them out like this?
+        noParse: (file: string) => file === join(stagingDir, 'l.js') || file === `./__ember_auto_import__/l.js`,
```

Comment:
Yes, and this would now *only* need to match the l.js in the virtual modules.

`this.stagingDir` should be able to be entirely refactored away.